src/page/product-list.page.ts

```diff
@@ -4,7 +4,7 @@ export class ProductListPage {
   private addToCart: ElementFinder;
 
   constructor () {
-    this.addToCart = $('#center_column a.button.ajax_add_to_cart_button.btn.btn-default');
+    this.addToCart = $('ul.product_list > li:nth-child(1) .button-container .ajax_add_to_cart_button');
```

Comment:
avoid using the selectors of type nth-child, they depend on the structure of the HTML, which can make the tests flaky, try to find an attribute or property that indicates we are clicking the add to cart button

Reply:
Done